src/multicellular_stage/editor/CellBodyPlanEditorComponent.GUI.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Godot;
+using Systems;
+
+/// <summary>
+///   The partial class containing GUI updating actions
+/// </summary>
+public partial class CellBodyPlanEditorComponent
+{
+    private readonly StringBuilder atpToolTipTextBuilder = new();
+
+    protected override void OnTranslationsChanged()
+    {
+        CalculateEnergyAndCompoundBalance(editedMicrobeCells);
+    }
+
+    private void ConfirmFinishEditingWithNegativeATPPressed()
+    {
+        if (OnFinish == null)
+        {
+            GD.PrintErr("Confirmed editing for cell editor when finish callback is not set");
+            return;
+        }
+
+        GUICommon.Instance.PlayButtonPressSound();
+
+        ignoredEditorWarnings.Add(EditorUserOverride.NotProducingEnoughATP);
+        OnFinish.Invoke(ignoredEditorWarnings);
+    }
+
+    private void HandleProcessList(IReadOnlyList<HexWithData<CellTemplate>> cells, EnergyBalanceInfo energyBalance,
+        IBiomeConditions biome)
+    {
+        // Empty list to later fill
+        var processStatistics = new List<ProcessSpeedInformation>();
+
+        var processes = new List<TweakedProcess>();
+
+        int cellCount = cells.Count;
+        for (int i = 0; i < cellCount; ++i)
+        {
+            ProcessSystem.ComputeActiveProcessList(cells[i].Data!.Organelles, ref processes);
```

Comment:
Isn't this code wrong? The active compute list will have items removed on each subsequent cell types that don't happen to run a process. Instead I think this code ought to be refactored to count how many of each cell type are in use and then run on each cell type and multiply it by the count to get to the final result.